src/unistd.rs

```diff
@@ -1132,6 +1132,57 @@ pub fn isatty(fd: RawFd) -> Result<bool> {
     }
 }
 
+
+
+/// Flags for `linkat` function.
+#[derive(Clone, Copy, Debug)]
+pub enum LinkatFlags {
+    SymlinkFollow,
+    NoSymlinkFollow,
+}
+
+/// Link one file to another file
+///
+/// Creates a new link (directory entry) at `newpath` for the existing file at `oldpath`. In the 
+/// case of a relative `oldpath`, the path is interpreted relative to the directory associated 
+/// with file descriptor `olddirfd` instead of the current working directory and similiarly for 
+/// `newpath` and file descriptor `newdirfd`. In case `flag` is LinkatFlags::SymlinkFollow and 
+/// `oldpath` names a symoblic link, a new link for the target of the symbolic link is created.
```

Comment:
The docs should mention what it means for `olddirfd` or `newdirfd` to be `None`.

Reply:
Hi ... didn't notice these comments. Will update the doc.

Reply:
Ok, Updated ... commented on the error conditions that are thrown when olddirfd or newdirfd is  set to None.